app/actions/configurations.py

```diff
@@ -39,23 +60,139 @@ class PullEventsConfig(PullActionConfiguration):
+                            "include_fire_alerts": {
+                                "const": True
+                            }
+                        }
+                    },
+                    "then": {
+                        "required": ["fire_lookback_days", "fire_alerts_lowest_confidence"],
+                        "properties": {
+                            "fire_lookback_days": {
+                                "type": "integer",
+                                "title": "Fire alerts lookback days",
+                                "default": 5,
+                                "maximum": 10,
+                                "minimum": 1,
+                                "description": "Number of days to look back for fire alerts."
+                            },
+                            "fire_alerts_lowest_confidence": {
+                                "allOf": [{
+                                    "$ref": "#/definitions/NasaViirsFireAlertConfidenceEnum"
+                                }],
+                                "title": "Fire alerts lowest confidence",
+                                "default": "high",
+                                "description": "Lowest confidence level to include in the connection."
+                            }
+                        }
+                    }
+                }, {
+                    "if": {
+                        "properties": {
+                            "include_integrated_alerts": {
+                                "const": True
+                            }
+                        }
+                    },
+                    "then": {
+                        "required": ["integrated_alerts_lookback_days", "integrated_alerts_lowest_confidence"],
+                        "properties": {
+                            "integrated_alerts_lookback_days": {
+                                "type": "integer",
+                                "title": "Integrated deforestation alerts lookback days",
+                                "default": 15,
+                                "maximum": 30,
+                                "minimum": 1,
+                                "description": "Number of days to look back for integrated deforestation alerts."
+                            },
+                            "integrated_alerts_lowest_confidence": {
+                                "allOf": [{
+                                    "$ref": "#/definitions/IntegratedAlertsConfidenceEnum"
+                                }],
+                                "title": "Integrated deforestation alerts lowest confidence",
+                                "default": "highest",
+                                "description": "Lowest confidence level to include in the connection."
+                            }
+                        }
+                    }
+                }]
+            })
+
+
+class GetDatasetAndGeostoresConfig(PullActionConfiguration):
```

Comment:
Subactions configs should inherit from `InternalActionConfiguration`, unless you want them to appear in the portal for user input.
```suggestion
class GetDatasetAndGeostoresConfig(InternalActionConfiguration):
```